lua/llm/config.lua

```diff
@@ -39,6 +39,16 @@ local default_config = {
   disable_url_path_completion = false,
 }
 
+local default_request_bodies = {
+  ollama = {
+    options = {
+      temperature = 0.2,
+      top_p = 0.95,
+    },
+  },
+  openai = {},
```

Comment:
Perhaps it'd be good to actually set some default params here, wdyt?

Reply:
Yeah that makes sense. I'll translate the defaults to OpenAI and add that to the README example as well.

Reply:
Done!

Reply:
perfect ty!